src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../styles/states.scss";
+@import "../../styles/global-css-settings.scss";
+
+@mixin toggle-animation {
+  transition: 100ms $expand-animation-timing;
+}
+
+.monday-style-toggle{
+  &__wrapper {
+    display: flex;
+    align-items: center;
+  }
+
+  &__toggle {
+    @include reset-button();
+    @include toggle-animation;
+    margin-inline: $spacing-small;
+    position: relative;
+    height: 24px;
+    width: 41px;
+    cursor: pointer;
+    border-radius: 100px;
```

Comment:
why 100px? is it supposed to be that?
there is also border-radius: 100%

Reply:
why 100px?